packages/ui-library/src/components/tooltip/index.stories.ts

```diff
@@ -97,84 +62,47 @@ export default {
       url: 'https://www.figma.com/file/C4vgEKz8mKyulJ4gm3Qdql/%F0%9F%AB%A7-%5BBLR%5D-The-B01LER?node-id=3618%3A126744&mode=dev',
     },
     viewMode: 'docs',
-    layout: 'padded',
+    layout: 'centered',
     docs: {
       description: {
         component: `
   <markdown>
-  Tooltip is triggered by hovering the mouse cursor over the element. It is designed to be unobtrusive and usually consist
-  of a short snippet of text. Tooltip component utilizes Floating UI library for granular positioning and interactions
-  based on user events.
+  Tooltip is triggered by hovering the mouse cursor over the element. It is designed to be unobtrusive and usually consist of a short snippet of text. To make this work technically the tooltip component can be combined with [Floating UI](https://floating-ui.com/) like shown on the example page [Tooltip using Floating UI](?path=/docs/examples-tooltip-using-floating-ui--docs)."
```

Comment:
Please remove the " at the end of the line

Reply:
Adjusted